### DISCUSSION TITLE
Azure private networking for GitHub-hosted runners

### DISCUSSION THREAD

**Discussion by krzysztof-madej** [community/community#101992] [Actions] [answered]
### Select Topic Area

Question

### Body

I have question regarding [Azure private networking for GitHub-hosted runners](https://github.blog/changelog/2023-11-01-github-hosted-runners-private-networking-with-azure-virtual-networks-public-beta/)

👉 Is this is/going to be available for regular runners or just for larger runners?
👉 Is there any progress on this? I mean other regions? 
👉 Is there any ETA when it would get GA status?

I'm asking these question because we want to move to GitHub and this feature will make transition much easier?

**Comment by ghost** [comment, marked as answer]
GitHub had been working on improving the integration with cloud providers for Actions, but specific details regarding availability for regular or larger runners, progress in other regions, and the timeline for General Availability (GA) status would be something GitHub would announce officially.

You can take to get the latest information from GitHub Public Roadmap: GitHub maintains a public roadmap on GitHub itself. This is a good place to check for any planned features, updates, or enhancements related to GitHub Actions and Azure integrations. Also you can find GitHub Changelog: The GitHub Changelog provides updates on all the latest features, bug fixes, and improvements. You might find information about Azure private networking there.

If this feature is critical for your organization's transition to GitHub, it would be best to directly contact GitHub for detailed information and possibly discuss your specific needs and timelines with them. They might be able to provide you with a beta access or more details on when you could expect such features to become available.
